tests/core/idelibrary/errors.livecodescript

```diff
@@ -17,7 +17,7 @@ You should have received a copy of the GNU General Public License
 along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
 
 constant kExtensionErrorCode = 863
-constant kExpectedError = "extension: error occured with domain"
```

Comment:
This change is dependent on executionerrors.h being modified as well.  It seems that this test was chosen as the first 8.0 specific error.  To avoid the "occured" to "occurred" changes in the .h, this test could be changed to 867 to make it independent of the updates to the .h file.